closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -2867,4 +2899,4 @@ Java_jdk_crypto_jniprovider_NativeCrypto_PBEDerive
     }
 
     return ret;
-}
+}
```

Comment:
Please restore the final newline here.

Reply:
updated

Reply:
I still don't see it. The final character in the file should be a newline, not the `}`.